packages/shared/src/utils/constants/mt5-login-list-status.ts

```diff
@@ -1,9 +1,24 @@
-export const MT5_ACCOUNT_STATUS = Object.freeze({
+/**
+ * this comes from mt5_login_list endpoint
+ */
+export const MT5_ACCOUNT_STATUS = {
     FAILED: 'failed',
     MIGRATED_WITH_POSITION: 'migrated_with_position',
     MIGRATED_WITHOUT_POSITION: 'migrated_without_position',
     NEEDS_VERIFICATION: 'needs_verification',
     PENDING: 'pending',
     POA_PENDING: 'poa_pending',
     POA_VERIFIED: 'poa_verified',
-});
+    UNDER_MAINTENANCE: 'under_maintenance',
+    UNAVAILABLE: 'unavailable',
+} as const;
+
+/**
+ * this comes from trading_platform_status endpoint
+ */
+export const TRADING_PLATFORM_STATUS = {
+    ACTIVE: 'active',
+    DISABLED: 'disabled',
+    MAINTENANCE: 'maintenance',
+    UNAVAILABLE: 'unavailable',
+} as const;
```

Comment:
I can see these values being hardcoded all over the code, just make use of this object that you defined.
<img width="255" alt="Screenshot 2024-07-22 at 3 26 37 PM" src="https://github.com/user-attachments/assets/4fbba60a-23ce-44ab-a406-6418474ac1ce">

Reply:
will add the constants